lib/alerts/components/AlertEditor.js

```diff
@@ -302,8 +308,16 @@ export default class AlertEditor extends Component<Props> {
                             : 'text-danger'
                         }
                         style={{fontWeight: 400}}>
-                        {descriptionCharactersRemaining}
+                        {descriptionCharactersCount}
                       </span>
+                        {descriptionCharactersCount > ALERT_DESCRIPTION_CHAR_LIMIT
+                          ? (
+                            <h5 style={{margin: '0px'}}>
+                              <small className='text-danger'>
+                                  {`WARNING:  Alert description longer than ${ALERT_DESCRIPTION_CHAR_LIMIT} characters may get truncated in some dissemination channels. `}
+                              </small>
+                            </h5>
+                          ) : ''}
```

Comment:
I personally would place the error message below the input box.

Same remark regarding https://react-bootstrap-v3.netlify.app/components/forms/#forms-validation, and it can probably accomplished without using a `<h5>` heading.